server/neptune/gateway/event/comment_handler.go

```diff
@@ -51,3 +75,88 @@ func (p *CommentEventWorkerProxy) Handle(ctx context.Context, request *http.Buff
 
 	return nil
 }
+
+func (p *CommentEventWorkerProxy) forceApply(ctx context.Context, event Comment) error {
+	rootCfgs, err := p.rootConfigBuilder.Build(ctx, event.HeadRepo, event.Pull.HeadBranch, event.Pull.HeadCommit, event.InstallationToken)
+	if err != nil {
+		return errors.Wrap(err, "generating roots")
+	}
+	for _, rootCfg := range rootCfgs {
+		p.logger.WarnContext(ctx, fmt.Sprintf("starting workflow"))
+		ctx = context.WithValue(ctx, contextInternal.ProjectKey, rootCfg.Name)
+		run, err := p.startWorkflow(ctx, event, rootCfg)
+		if err != nil {
+			return errors.Wrap(err, "signalling workflow")
+		}
+
+		p.logger.InfoContext(ctx, "Signaled workflow.", map[string]interface{}{
+			"workflow-id": run.GetID(), "run-id": run.GetRunID(),
+		})
+	}
+	return nil
+}
+
+func (p *CommentEventWorkerProxy) startWorkflow(ctx context.Context, event Comment, rootCfg *valid.MergedProjectCfg) (client.WorkflowRun, error) {
```

Comment:
I thought about that too, but I figured we should wait a little bit more to make sure we are more stable with our current workflow request definitions before making that change. I can add a TODO for it.

Reply:
I mean, i think less stable means its better to have it one location because it's only one place to change.  I'd rather we just do it as part of this.